packages/valory/skills/abstract_round_abci/tests/test_tools/test_common.py

```diff
@@ -0,0 +1,166 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Tests for abstract_round_abci/test_tools/common.py"""
+
+from pathlib import Path
+from typing import Type, cast
+from unittest import mock
+
+import pytest
+from aea.components.base import _CheckUsedDependencies  # for temporary patch
+from aea.helpers.base import cd
+
+from packages.valory.skills.abstract_round_abci.test_tools.common import (
+    BaseRandomnessBehaviourTest,
+)
+from packages.valory.skills.transaction_settlement_abci.behaviours import (
+    RandomnessTransactionSubmissionBehaviour,
+    SelectKeeperTransactionSubmissionBehaviourA,
+)
+
+# need to change
+from packages.valory.skills.transaction_settlement_abci.rounds import Event
+
+
+test_skill = "transaction_settlement_abci"
+PATH_TO_SKILL = BaseRandomnessBehaviourTest.path_to_skill.parent / test_skill
```

Comment:
replaced in next #1494 . Wanted to get tests working before constructing a dummy abci